src/main/scala/firrtl/stage/FirrtlAnnotations.scala

```diff
@@ -177,6 +177,9 @@ case class RunFirrtlTransformAnnotation(transform: Transform) extends NoTargetAn
 
 object RunFirrtlTransformAnnotation extends HasShellOptions {
 
+  def apply(transform: TransformDependency): RunFirrtlTransformAnnotation =
+    RunFirrtlTransformAnnotation(transform.getObject())
```

Comment:
```suggestion
    RunFirrtlTransformAnnotation(transform.getObject)
```
Nit, but parentheses on 0-arity methods implies mutation

Reply:
The reason why this happened is that the method is defined with parenthesis: https://github.com/freechipsproject/firrtl/blob/564312c3a813498b3ba5b88c6984b9cbeb94dd12/src/main/scala/firrtl/options/Phase.scala#L42

Reply:
Yeah that was a mistake, but deleting those parentheses is a backwards incompatible API change whereas at least not propagating the mistake is free :)